python/rmm/docs/guide.md

```diff
@@ -67,6 +67,38 @@ host:
 array([1., 2., 3.])
 ```
 
+## Prefetching DeviceBuffers Backed by Managed Memory
+
+CUDA Unified Memory, also known as managed memory, can be allocated using
+rmm.ManagedMemoryResource explicitly, or by calling `rmm.reinitialize` with
+`managed_memory=True`.
```

Comment:
```suggestion
CUDA Unified Memory, also known as managed memory, can be allocated using an
`rmm.mr.ManagedMemoryResource` explicitly, or by calling `rmm.reinitialize` with
`managed_memory=True`.
```

Reply:
*question*: Shall we link "CUDA Unified Memory` anywhere? Perhaps https://developer.nvidia.com/blog/unified-memory-cuda-beginners/ ?

Reply:
Add this link, and one for HMM.